www/src/components/repository/RepositoryTests.js

```diff
@@ -170,28 +169,35 @@ function TestDetail({ test, setTest }) {
   const len = test.steps.length
 
   return (
-    <Box fill>
-      <Box
-        flex={false}
-        direction="row"
-        gap="small"
+    <>
+      <Flex
         align="center"
-        pad="small"
-        background="fill-one"
-        border
+        border="1px solid border"
+        backgroundColor="fill-one"
+        paddingVertical="small"
+        paddingHorizontal="medium"
+        marginBottom="medium"
+        borderRadius="large"
       >
         <Icon
-          icon={<Return size="15px" />}
+          icon={(
+            <Return size="15px" />
+          )}
           onClick={() => setTest(null)}
         />
-        <Span fontWeight="bold">{test.name}</Span>
-      </Box>
+        <Span
+          bold
+          marginLeft="medium"
+        >
+          {test.name}
+        </Span>
+      </Flex>
       <Table
         headers={['Name', 'Description', 'Last Updated', 'Status']}
         sizes={['25%', '25%', '25%', '25%']}
         background="fill-one"
         width="100%"
-        height="100%"
+        height="calc(100% - 16px - 16px - 53px)" // The previous node is 53px tall with a 16px marginBottom
```

Comment:
Wouldn't it be easier to put this in a 100%x100% box with 16px vertical padding?

Reply:
Does work with flex layouts IMO, but if you know better please commit coz I ain't fighting this battle again.